### ISSUE TITLE
Distribution/Compilation Option

### ISSUE THREAD

**Issue by dbready** [fubark/cyber#46]
Will there be a first-party way or producing a single binary for distributing code? With the focus on embeddibility, I assume it should be easy enough to make a minimal wrapping application, but would love to see this available from the native tooling. 

As a Python programmer, distributing code is still needlessly painful with a multitude of partially working options. Would love if Cyber had a solid story here, even if it only worked with pure Cyber code.

**Comment by fubark** [comment]
Definitely, Cyber will do this. For desktop, it'll do just as you said, which is wrapping the scripts inside a copy of the VM which is incredibly small (under 2MB). To skip compilation, the bytecode could be embedded as well into the binary.

For Web/WASM, it can do something similar and package it into one .wasm file. We could also target WASM directly more easily than other GC languages, since Cyber uses ARC.

There's also the possibility that Cyber could target a native language in the future... Maybe C or Zig.
